schema/compose-spec.json

```diff
@@ -500,9 +500,7 @@
             },
             "additionalProperties": false,
             "patternProperties": {"^x-": {}}
-          },
-          "additionalProperties": false,
-          "patternProperties": {"^x-": {}}
```

Comment:
Should be moved to the `development` level as this section is missing the capability to add extension attributes `x-`

Reply:
Added them back on development